app/models/business_day.rb

```diff
@@ -17,6 +17,10 @@ def business_days_since_day_zero
     calendar.business_days_between(day_zero, Time.current.in_time_zone('London').to_date)
   end
 
+  def starts_on
+    calendar.subtract_business_days(date, 1).tomorrow
```

Comment:
Why are we subtracting only one day? can we create a local variable with more of a descriptive name.

Reply:
`weekend_or_bankholiday_offset = 1`